File: problems/linked_lists/createLinkedList.js
const createLinkedList = (values) => {
  if(!values.length) return null 
  let head;
  let prev = null 
  for(let i = 0; i < values.length; i++) {
    let newNode = new Node(values[i])
    if(i === 0) head = newNode
    if(i !==0) prev.next = newNode 
    prev = newNode 
  }
  return head 
};

File: python_Structy/create_linked_list.py
class Node:
  def __init__(self, val):
    self.val = val
    self.next = None

def create_linked_list(values, i = 0):
  if i == len(values): return None
  node = Node(values[i])
  node.next = create_linked_list(values, i+1)
  return node